core/themes/seven/css/style.css

```diff
@@ -789,16 +789,11 @@ input.form-search,
 input.form-file,
 textarea.form-textarea,
 select.form-select {
```

Comment:
It looks like these changes are affecting the select element as well.  Select element styles should be handled in https://github.com/backdrop/backdrop-issues/issues/754